package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@signalk/n2k-signalk",
-  "version": "1.3.0",
+  "version": "1.3.1",
```

Comment:
I don't think it makes sense to bump the version in a PR, as you can know in which version the changes will be published.